webapp-django/crashstats/settings/base.py

```diff
@@ -654,8 +654,8 @@ def path(*dirs):
         },
         'boto': {
             'access_key': config('resource.boto.access_key', None),
-            'bucket_name': config(
-                'resource.boto.bucket_name', 'crashstats'),
+            'bucket_name': config('resource.boto.bucket_name', 'crashstats'),
+            'region': config('resource.boto.region', 'us-west-2'),
```

Comment:
This moves `bucket_name` to one line which is cosmetic. This also adds `region` using the default from the connection class. Otherwise we have no way of changing the region via env vars in the webapp.